ChatQnA/docker_compose/intel/cpu/xeon/compose_qdrant.yaml

```diff
@@ -53,6 +53,7 @@ services:
       QDRANT_HOST: ${QDRANT_HOST}
       QDRANT_PORT: 6333
       INDEX_NAME: ${INDEX_NAME}
+      TEI_EMBEDDING_ENDPOINT: ${TEI_EMBEDDING_ENDPOINT}
```

Comment:
https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/qdrant/langchain/config.py#L17

Reply:
https://github.com/bigclouds/GenAIExamples/blob/main/ChatQnA/docker_compose/intel/cpu/xeon/README_qdrant.md#1-build-retriever-image    ; this link tells how  retriever-qdrant:latest image is build.

comps/retrievers/qdrant/haystack/qdrant_config.py is to get variable from env. 
EMBED_ENDPOINT = os.getenv("TEI_EMBEDDING_ENDPOINT", "")